### ISSUE TITLE
Remove ignoreDirection from teleport effect

### ISSUE THREAD

**Issue by bluelhf** [SkriptLang/Skript#3050]
### Description
The ignoring of cardinal directions by the teleport effect confuses a lot of people, from what I've seen.
Are there any reasons why it should exist? Wouldn't it be better to have the end-user explicitly specify that they want to keep the yaw and pitch of the player, instead of creating a confusing situation where a yaw of `0.001` teleports the player correctly (including yaw), but a yaw of `0` uses the old yaw of the player?

TL;DR the ignoreDirection method in the teleport effect is seemingly useless and confusing, admin pls fix

**Comment by APickledWalrus** [comment]
Fixed in #3123 
Should be in the next release!
